doc/source/whatsnew/v0.21.0.txt

```diff
@@ -40,7 +40,7 @@ Other Enhancements
 - :func:`DataFrame.clip()` and :func:`Series.clip()` have gained an ``inplace`` argument. (:issue:`15388`)
 - :func:`crosstab` has gained a ``margins_name`` parameter to define the name of the row / column that will contain the totals when ``margins=True``. (:issue:`15972`)
 - :func:`Dataframe.select_dtypes` now accepts scalar values for include/exclude as well as list-like. (:issue:`16855`)
-
+- Improved thread safety for `read_html()`. (:issue:`16928`)
```

Comment:
I'm actually inclined to call this a bug because multi-threading is something that we check for unless explicitly stated otherwise.  Thus, I would move to this to the bugs section.  Also, I would expand your description about the issue was (just a sentence but a longer one would suffice).